### DISCUSSION TITLE
Is it possible to set request limit from unique IP addresses?

### DISCUSSION THREAD

**Discussion by aviaku** [Web3Auth/Web3Auth#818] [Help]
Hello guys,
Thanks in advance

I'm integrating web3auth in my DAPP with unity. I want to limit the web3auth request for users. Is this possible???

**Comment by shahbaz17** [comment]
Hey @aviaku 

Can you add more context to `I want to limit the web3auth request for users.`? How would that look?

**Reply by aviaku** [reply]
Ya, I want a user to be able to make only 20 login requests per minute from one IP address.
